src/components/new-request/business-fetch.vue

```diff
@@ -28,12 +31,7 @@ import Vue from 'vue'
 import { Component, Emit } from 'vue-property-decorator'
 import { Action } from 'vuex-class'
 import { BusinessSearchIF } from '@/interfaces'
-
-enum States {
-  INITIAL = 'initial',
-  SEARCHING = 'searching',
-  SUMMARY = 'summary'
-}
```

Comment:
TS complained about this so I moved it into a new file in enums folder.

Reply:
Out of curiosity, what was TS complaining about exactly Sev? That enums have to be separate and not be defined in the components?

Reply:
It said they were "unused", even though I was clearly using them 🤷‍♂️

Reply:
Well that's weird, haha. All right Sev.